templates/user/dashboard/feeds.tmpl

```diff
@@ -73,10 +73,14 @@
 					{{else if .GetOpType.InActions "publish_release"}}
 						{{$linkText := .Content | ctx.RenderUtils.RenderEmoji}}
 						{{ctx.Locale.Tr "action.publish_release" (.GetRepoLink ctx) (printf "%s/releases/tag/%s" (.GetRepoLink ctx) .GetTag) (.ShortRepoPath ctx) $linkText}}
-					{{else if .GetOpType.InActions "review_dismissed"}}
+					{{else if .GetOpType.InActions "pull_review_dismissed"}}
 						{{$index := index .GetIssueInfos 0}}
 						{{$reviewer := index .GetIssueInfos 1}}
 						{{ctx.Locale.Tr "action.review_dismissed" (printf "%s/pulls/%s" (.GetRepoLink ctx) $index) $index (.ShortRepoPath ctx) $reviewer}}
+					{{else if .GetOpType.InActions "auto_merge_pull_request"}}
+						{{$index := index .GetIssueInfos 0}}
+						{{$title := index .GetIssueInfos 1}}
+						{{ctx.Locale.Tr "action.auto_merge_pull_request" (printf "%s/pulls/%s" (.GetRepoLink ctx) $index) $index (.ShortRepoPath ctx) $title}}
```

Comment:
Just copied and pasted the code without checking it?


`auto_merge_pull_request = `automatically merged pull request <a href="%[1]s">%[3]s#%[2]s</a>``


But you put 4 arguments here?

Reply:
799ac4fb308eedffa65c2b55263bd0651fa5f59b The last argument `$title` should be removed.

Reply:
I do not see how really you fixed the copy-paste code.

If `$title` should be removed, why there is still `{{$title := index .GetIssueInfos 1}}` in the new commit?

Reply:
361d72f